### PR TITLE
PHPUnit 8 - test-legacy.php.php

### PR DESCRIPTION
**Pull request by eileenmcnaughton**

### CODE REVIEW

**Comment by seamuslee001** [comment]
LGTM @totten

File: src/CRM/CivixBundle/Resources/views/Code/test-legacy.php.php [hunk 1]
```diff
@@ -22,33 +22,42 @@
...
-  }
+  /**
+   * Setup for when Headless interface is implemented.
+   *
+   * `CiviTestListener`+`HeadlessInterface` has some clever tricks for
+   * bootstrapping which make it easier to use phpunit CLI.
+   * However, there's not much point in using setupHeadless() with CiviUnitTestCase
+   * because CiviUnitTestCase performs its own special setup/teardown logic.
+   */
+  public function setUpHeadless(): void {}
```
Comment (totten) [inline src/CRM/CivixBundle/Resources/views/Code/test-legacy.php.php:33]: Doesn't make sense on a few fronts:

* This disagrees with `HeadlessInterface`'s signature for `setUpHeadless()`
* Other templates have it return a value...

**Comment by eileenmcnaughton** [comment]
I'm just gonna close this - people shouldn't be creating new legacy tests

**Comment by totten** [comment]
I bumped into this incidentally -- there were some failures when running `make-example.sh` on master. With this, it did seem to pass. So I've cherry-picked and pushed.

**Pull request closed without merging**